pkg/NAMESPACE

```diff
@@ -65,4 +65,5 @@ export(
        "setCheckpointDir"
       )
 export("sparkR.init")
+export("print.jobj")
```

Comment:
If we don't export it, it can't be used from the shell. The shell by default calls `print` when you just type in a variable. For example, without this export after creating a SparkContext, I get
```
> sc
<environment: 0x7fbc2306c0a8>
attr(,"class")
[1] "jobj"
```

After this export, I get
```
> sc
Java ref type org.apache.spark.api.java.JavaSparkContext id 0
```